utils/generateExitCodesFiles.py

```diff
@@ -109,27 +109,24 @@ def genNSISFile(errors, filename):
 
 # generate C header file from errors
 def genCHeaderFile(errors):
-  wiki = "/// Exit codes and meanings\n"\
-       + "/// The following codes might be returned when exiting ArangoDB:\n"
+  wiki = "// Exit codes and meanings\n"\
+       + "// The following codes might be returned when exiting ArangoDB:\n"
 
   header =   "#pragma once\n"\
            + "\n"\
            + "#include \"Basics/error.h\"\n"\
            + "\n"\
-           + wiki\
-           + "\n"
+           + "/* clang-format off */\n"\
+           + wiki
 
   # print individual errors
   for e in errors:
     header = header\
+           + "\n"\
            + "/// " + e[1] + ": " + e[0] + "\n"\
            + wrap(e[2], 80, 0, 0, "/// ") + "\n"\
            + wrap(e[3], 80, 0, 0, "/// ") + "\n"\
```

Comment:
Why does clang-format not complain about `///` here?

Reply:
@Simran-B : please leave the PR alone. I already added a fix for clang-format complaining about it.